CodeStarterCamp_Week3/main.swift

```diff
@@ -12,46 +12,56 @@ struct Person {
     let name: String
     var age: Int
     var money: Int
-
-    mutating func buyCoffee(_ coffee: Coffee) {
+        
+    mutating func order(_ coffee: Coffee, of coffeeShop: CoffeeShop, by name: String) {
         print(coffee.rawValue + " 주세요.")
         
-        if let coffeePrice = yagombucks.menu[coffee] {
-            self.money -= coffeePrice
-            
-            print(name + "의 남은 돈:\(money)\n")
-        } else {
-            print("nil")
+        guard let coffeePrice = coffeeShop.menu[coffee] else { return }
+            if coffeePrice <= self.money {
+                self.money -= coffeePrice
+                print(name + "의 남은 돈:\(money)\n")
+                
+                coffeeShop.make(coffee, for: name)
+            } else {
+             print("잔액이 \(-(money - coffeePrice))원만큼 부족합니다.\n")
```

Comment:
```suggestion
             print("잔액이 \(coffeePrice - money)원만큼 부족합니다. \n")
```
요렇게 바꿔도 값이 잘 나올 거 같네요!